docs/src/main/sphinx/glossary.rst

```diff
@@ -54,8 +54,8 @@ Container
 .. _glossDataSource:
 
 Data source
-    A system from which data is retrieved, for example, PostgreSQL or Iceberg on
-    S3 data. In Trino, users query data sources with :ref:`catalogs
+    A system from which data is retrieved - for example, PostgreSQL. or Iceberg
```

Comment:
The period here is incorrect and makes the sentence after a fragment.

Reply:
oopsie